css-canvas-medical/styles.css

```diff
@@ -0,0 +1,262 @@
+* {
+  line-height: 30px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  margin: auto;
+}
+
+main {
+  margin-top: 75px !important;
+}
+
+.nav-follow {
+  position: fixed;
+  z-index: 2;
+  background-color: white;
+  border-bottom: solid #6e6e6e 1px;
+  top: 0;
+}
+
+.margin-side-only {
+  margin: 0.75rem 10% 0;
+}
+
+.margin {
+  margin: 2rem 10%;
+}
+
+.background-img {
+  width: 85%;
+  height: 600px;
+  background-size: cover;
+  position: relative;
+  background-image: url("images/hero.jpg");
+  margin: 0 10%;
+  background-position: center;
+}
+
+.background-img > img {
+  width: 85%;
+}
+
+.col-full {
+  flex-basis: 100%;
+}
+
+.col-1-2 {
+  flex-basis: 50%;
+}
+
+.col-1-3 {
+  flex-basis: calc(100% / 3);
+}
+
+.col-1-4 {
+  flex-basis: 25%;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+header {
+  border-bottom: solid gray 1px;
+}
+
+.image-overlay {
+  color: white;
+  z-index: 1;
+  padding: 7% 40% 10% 5%;
+}
+
+.image-fog {
+  background: #52535166;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
+
+.flex {
+  display: flex;
+}
+
+.wrap {
+  flex-wrap: wrap;
+}
+
+.no-wrap {
+  flex-wrap: nowrap;
+}
+
+.dir-col {
+  flex-direction: column;
```

Comment:
This doesn't seem to be doing anything as far as I can tell, because it's not being used alongside anything with `display: flex`. It's fine to create a utility class like this, but if you don't end up using it, I would remove it.